advanced/sr_gui_bootloader/src/sr_gui_bootloader/bootloader.py

```diff
@@ -13,20 +13,17 @@
 # with this program. If not, see <http://www.gnu.org/licenses/>.
 
 from __future__ import absolute_import
```

Comment:
To be fair we don't need this anymore and it would even be better to remove it as it can cause unexpected issues. For instance, if someone has a file called sys.py then we will not be able to call the real sys library...